domains/sonom.json

```diff
@@ -0,0 +1,12 @@
+{
+    "username": "getSono",
+    "email": "julianwolf2013@outlook.de",
+    "discord": "1222250424309121145",
```

Comment:
these need to be in the owner structure (check here https://www.is-a.dev/docs/domain_structure/)